views/madecharts.mustache

```diff
@@ -0,0 +1,45 @@
+<html>
```

Comment:
This should probably not be in this PR. This PR is just to add a unified header and footer to the _existing pages_